department-of-veterans-affairs/caseflow
# feature/APPEALS-32044 | Individual Change History

Jira: https://jira.devops.va.gov/browse/APPEALS-32044

# Description
As a Caseflow VHA Admin User, I need to view change history records associated to a claim in order to identify errors, keep accurate records of changes, and generate metrics on Caseflow use. 

## Acceptance Criteria
- [ ] The existing Generate Report History continues to work as expected
- [ ] Users can now see the Individual Claim History changes by navigating to `/decision_reviews/vha/tasks/{task_id}/history`

## Testing Plan
1. Navigate to the `/decision_reviews/vha/tasks/{task_id}/history` page as a VHA Admin.
2. Ensure that the claim history loads and is displayed

# Frontend
## User Facing Changes
![Screenshot 2024-03-18 at 12 51 02 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/51007432/d6bddd2b-2383-44a8-bd53-5c2dfdd9ef67)


## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
